xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -489,9 +489,9 @@
         <ReturnType>System.Security.Cryptography.X509Certificates.X509ChainTrustMode</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Mode determining root trust for building certificate chain.</summary>
+        <value>X509ChainTrustMode value determining root trust.</value>
+        <remarks>When <see cref="P:System.Security.Cryptography.X509Certificates.X509ChainTrustMode.CustomRootTrust" /> is used the <see cref="P:System.Security.Cryptography.X509Certificates.X509ChainPolicy.CustomTrustStore" /> will be used instead of default root trust.</remarks>
```

Comment:
```suggestion
        <summary>The mode determining the root trust for building the certificate chain.</summary>
        <value>An <see cref="T:System.Security.Cryptography.X509Certificates.X509ChainTrustMode" /> value determining the root trust.</value>
        <remarks>
          <format type="text/markdown"><[!CDATA[

## Remarks

When <xref:System.Security.Cryptography.X509Certificates.X509ChainTrustMode.CustomRootTrust> is used, the <xref:System.Security.Cryptography.X509Certificates.X509ChainPolicy.CustomTrustStore> will be used instead of the default root trust.

        ]]></format>
        </remarks>
```